sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -2046,10 +2046,15 @@ pi_result piextUSMGetMemAllocInfo(pi_context, const void *, pi_mem_alloc_info,
 }
 
 pi_result piextUSMImport(const void *ptr, size_t size, pi_context context) {
+  (void)ptr;
+  (void)size;
+  (void)context;
   return PI_SUCCESS;
 }
```

Comment:
Just a comment: another/simpler way to handle unused parameters is not giving them a name, which work too
```suggestion
pi_result piextUSMImport(const void *, size_t, pi_context) {
  return PI_SUCCESS;
}
```